### PR TITLE
Warn on DBC signal name usage across multiple messages

### PR DESCRIPTION
**Pull request by sophokles73**
During startup, the DBC Feeder reads in CAN message definitions
from arbitrary database files. When using the same signal name in
different message definitions, care needs to be taken that the semantics
of the signal are the same across all messages.

The DBC Feeder has been changed to detect such cases and issue a
corresponding warning if the signal is being mapped to a VSS datapoint.

This addresses #122

### CODE REVIEW

File: dbc2val/dbcfeeder.py [hunk 1]
```diff
@@ -131,18 +132,18 @@ def __init__(self, client_wrapper: clientwrapper.ClientWrapper,
...
-        dbcfile,
-        mappingfile,
-        dbc_default_file,
-        candumpfile=None,
-        use_j1939=False,
-        use_strict_parsing=False
+        dbcfile: str,
+        mappingfile: str,
+        dbc_default_file: str,
+        candumpfile: str = None,
```
Comment (erikbosch) [inline dbc2val/dbcfeeder.py:138]: Mypy check fails in CI, here I believe that it want `optional[str]` as typehint

```
 dbcfeeder.py:138: error: Incompatible default for argument "candumpfile" (default has type "None", argument has type "str")  [assignment]
dbcfeeder.py:138: note: PEP 484 prohibits implicit Optional. Accordingly, mypy has changed its default to no_implicit_optional=True
```

**Comment by erikbosch** [comment]
Conflicts now as well

**Comment by erikbosch** [comment]
@sophokles73 - I think it would be bad if we would loose this PR. Do you intend to work something on it within the next months, or should I try to do some work to make it up-to-date for the new repo?

**Comment by sophokles73** [comment]
@erikbosch let me see if I find the time to migrate the PR to the new repo ...

**Comment by sophokles73** [comment]
@erikbosch I have rebased this PR locally. Once #122 has been migrated to the new repo, I will recreate the PR in the new repo ...

**Comment by erikbosch** [comment]
I created a new issue in the new repo, see https://github.com/eclipse-kuksa/kuksa-can-provider/issues/8

**Pull request closed without merging**